Packs/Darktrace/Integrations/DarktraceEventCollector/README.md

```diff
@@ -0,0 +1,46 @@
+Use this integration to fetch a list of model breaches, filtered by the specified parameters. This is important for organizations that wish to integrate Darktrace programmatically into their SOC environment.
```

Comment:
please add the missing "This integration was integrated and tested with version... of Darktrace"